### PR TITLE
feat(mode): adding native component pairs

### PR DESCRIPTION
**Pull request by theJohnnyMe**
Adding missing native pairs of components.
I did commit per commit so you can easily follow up what was added or changed, please refer to it if possible.

### CODE REVIEW

File: react-cra/src/pages/WebComponents.tsx [hunk 1]
```diff
@@ -273,6 +273,27 @@ export default function WebComponents() {
...
         </sdds-modal>
       </div>
+
+      <div>
+        <div className="sdds-headline-02 sdds-u-pb1">Radio Button</div>
+        <sdds-radio-button
+            name="rb-example"
+            value="option1"
+            radio-id="option-1"
+            checked
```
Comment (mJarsater) [inline react-cra/src/pages/WebComponents.tsx:283]: No warning on this one?

**Comment by mJarsater** [comment]
There are some lint error that's crashing the build. Either we change the netlify config or we fix it?

File: react-cra/src/pages/WebComponents.tsx [hunk 2]
```diff
@@ -131,8 +131,8 @@ export default function WebComponents() {
...
-          body-img="https://scdn.reflex.prod.aws.scania.com/thumbnails/2.6a4c8c88169b49c419d1a8/18.12bc286418679e166a96d74.webp?Expires=1678251771&Key-Pair-Id=K2MBMFK028C481&Signature=Zq51cDQl-92BG136TnDH57Lf46CXNdlI2WAIfLmg~NsYqITvoTMtCJpb5wJLPEMmDY~ufJMbJmaKoMo93yZVZSXihFehAk40slf8v40czQ1uMydC-85vYlItEy~5cSeu8wsLWRXyUCTv7BDlHQ2OWxFXjv0QMR2ggoHb1cBsElWshMILTm69VwrMtKutfGQz9~cLG0ApjeyOPmTBMbI4SU0FexiNCDUmjWAuLGvj2I02ExxetbUQU8pcddMgnsnfubRCzNpK2b~pQBNGIA4yYjqywPElVGQG0B7Xk8J9XY70gQRYXVZph6sT-umLb95TGPymRZoczmFgtzxg4xYHqg__"
+          header-img="https://images.unsplash.com/photo-1499887142886-791eca5918cd?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1160&q=80  "
+          body-img="https://images.unsplash.com/photo-1670509295484-df0c2512fec4?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=5494&q=80"
```
[the remark this answers is not in the document]
Comment (theJohnnyMe) [reply react-cra/src/pages/WebComponents.tsx:135]: I was annoyed with contact image braking, this should be more stable.

**Comment by theJohnnyMe** [comment]
> There are some lint error that's crashing the build. Either we change the netlify config or we fix it?

Should be fixed now ;) I think it is good for us, it makes us take care of details too.

**Review by mJarsater — APPROVED** [review]
🥫 <- Awesome sauce

File: react-cra/src/pages/WebComponents.tsx (hunk 1 above)
Comment (theJohnnyMe) [reply react-cra/src/pages/WebComponents.tsx:283]: Screenshot 2023-03-10 at 14 17 43
 Just a value missing warning, like with any prop that is boolean...

**Pull request merged**